test/e2e/specs/ExistingUserCreditCheckout.spec.js

```diff
@@ -0,0 +1,55 @@
+describe('Existing User Checkout with Kiva Credit', () => {
+	it('Baskets a loan, proceed to checkout, log in user with credit and purchases using kiva credit', () => {
+		let beginBalance;
+		let newBalance;
+		// get the test user id from cypress.env.json and set the cookie variable
+		const lenderID = Cypress.env().userInfo.kivaCreditLender.userID;
+		const loginCookie = `${lenderID}:recent/active/mfa`;
+
+		// Mock login our test lender
+		cy.setCookie('kvfa', loginCookie, 'domain=.kiva.org');
+		// go to portfolio and get current credit for later calculations
+		cy.visit('/portfolio');
+		cy.get('.strong .kiva-green').invoke('text').then(fullText => {
+			const number = fullText.match(/[0-9,.]+/);
+			beginBalance = parseFloat(number); // text to float
+		});
+		// Go to lend page
+		cy.visit('/lend?x');
```

Comment:
Can we use the loan carousel on the `/checkout` page to add a loan instead of adding a dependency on the legacy stack being present?

If not, some other alternatives would be to use a Ui app category page such as `/lend-by-category/agriculture`

Reply:
Okay, I'll make a change to use on category page

Reply:
Oh yea, good call on test ids. None of the lend buttons currently have data-testid attributes :(